docs/generated/oc_by_example_content.adoc

```diff
@@ -920,6 +933,32 @@ Create a service account with the specified name.
 ====
 
 
+== oc create user
+Create a user.
+
+====
+
+[options="nowrap"]
+----
+  # Create a user with the username "ajones" and the display name "Adam Jones"
+  $ oc create user ajones --full-name="Adam Jones"
+----
+====
+
+
+== oc create useridentitymapping
```

Comment:
Understood. Let's be clear in the help text about when/why you'd use these tools. Otherwise code LGTM. Lots of boilerplate for `create`, but I'm not really sure if we want or need a better abstraction.